modules/KoreBuild.Tasks/MergeXmlFiles.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.AspNetCore.BuildTools.Utilities;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace KoreBuild.Tasks
+{
+    public class MergeXmlFiles : Task
+    {
+        [Required]
+        public string OutputPath { get; set; }
+
+        [Required]
+        public ITaskItem[] Files { get; set; }
+
+        public override bool Execute()
+        {
+            if (Files == null || Files.Length == 0)
+            {
+                Log.LogError("No files could be found to merge");
+                return false;
+            }
+
+            OutputPath = FileHelpers.NormalizePath(OutputPath);
+            Directory.CreateDirectory(Path.GetDirectoryName(OutputPath));
+
+            if (Files.Length == 1)
+            {
+                File.Copy(Files[0].ItemSpec, OutputPath);
+                return true;
+            }
+
+            return Execute(() => File.CreateText(OutputPath));
+        }
+
+        protected internal bool Execute(Func<TextWriter> writerFactory)
```

Comment:
I assume this task has a "Be it on your own head if this merge results in duplicate results." warning on it since we're not doing and explicit deduping.

Reply:
correct. In aspnet/Universe, if we had conflicting files, we would have other issues, and not just in the xml.